fusor-ember-cli/app/components/tr-subscription.js

```diff
@@ -26,7 +26,7 @@ export default Ember.Component.extend({
   }),
 
   isQtyValid: Ember.computed('subscription.qtyAvailable', 'subscription.qtyToAttach', function() {
-    if ((this.get('subscription.qtyToAttach') > 0) && (this.get('subscription.qtyAvailable') > 0)) {
+    if ((this.get('subscription.qtyToAttach') >= 0) && (this.get('subscription.qtyAvailable') > 0)) {
```

Comment:
fixed small bug, previously entering 0 was "invalid" changed is to `qtyAvailable` but now it's only if the the qty entered is greater than what's available.